nsxt/resource_nsxt_policy_project.go

```diff
@@ -270,9 +270,17 @@ func patchVpcSecurityProfile(d *schema.ResourceData, connector client.Connector,
 			}
 		}
 	}
+
+	clientSecurity := projects.NewVpcSecurityProfilesClient(connector)
+	objSec, err := clientSecurity.Get(defaultOrgID, projectID, "default")
+	if isNotFoundError(err) {
+		return fmt.Errorf("failed to fetch the details of the VPC security profile: %v", err)
+	}
 	// Default security profile is created by NSX, we can assume that it's there already
 	client := projects.NewVpcSecurityProfilesClient(connector)
```

Comment:
I don't think you need to reinstantiate the client here, you can reuse the same one you instantiated at line 274